tests/components/dwd_weather_warnings/test_init.py

```diff
@@ -25,13 +25,11 @@ async def test_load_unload_entry(
     entry = await init_integration(hass, mock_identifier_entry)
 
     assert entry.state is ConfigEntryState.LOADED
-    assert entry.entry_id in hass.data[DOMAIN]
```

Comment:
Optional. Do you want to add an assert here that the coordinator is stored inside the `entry.runtime_data`?

Something like:
```py
assert isinstance(entry.runtime_data, DwdWeatherWarningsConfigEntry)

# or simpler (less type safe though)
assert entry.runtime_data is not None
```

Reply:
sounds reasonable 👍